### ISSUE TITLE
On iPad, can't click on "rules"

### ISSUE THREAD

**Issue by phunknsoul** [robx/pzprjs#205]
As title says, on my iPad (iPad2?  I do realize it's an older model if that makes a difference), When I click on Help, then try to click Rules, but nothing happens.  It doesn't turn grey to indicate that it's being pressed like the other menu items.  All other menu items work fine (except Bug Reports and Feature Requests).  The Help works fine on my Android phone and my PC... it's only the iPad that has this problem.

**Comment by robx** [comment]
Yeah, I tried digging into this before but failed to find a fix (or I thought I had a fix?). Thanks for filing the issue!
